diff --git a/node_modules/react-joyride/dist/index.mjs b/node_modules/react-joyride/dist/index.mjs
index 7384a27..14352f2 100644
--- a/node_modules/react-joyride/dist/index.mjs
+++ b/node_modules/react-joyride/dist/index.mjs
@@ -1,6 +1,6 @@
 import e, { PureComponent as t, Component as r, useState as n, useEffect as o, useLayoutEffect as i, useRef as s, useCallback as a } from "react";
-import { createPortal as c, findDOMNode as u, unmountComponentAtNode as l, unstable_renderSubtreeIntoContainer as p } from "react-dom";
 import d from "react-is";
+import { createPortal as c, findDOMNode as u } from "react-dom";
 import m from "deepmerge";
 import h from "hoist-non-react-statics";
 import f from "is-plain-object";
@@ -459,7 +459,7 @@
   }
   componentWillUnmount() {
     this.portal && (0, o.unmount)(this.portal);
-  }
+  } 
   render() {
     const {
       props: {
@@ -522,7 +522,7 @@
     this.node.style.position = "absolute", this.node.style.top = "0", this.node.style.width = "100vw", this.node.style.zIndex = z, t.appendChild(this.node);
   }
   unmount() {
-    this.isReact16 ? l(this.node) : p(null, this.node, () => {
+    this.isReact16 ? unmountComponentAtNode(this.node) : unstable_renderSubtreeIntoContainer(null, this.node, () => {
       this.node.parentNode.removeChild(this.node);
     });
   }
